### DISCUSSION TITLE
Excel window size

### DISCUSSION THREAD

**Discussion by AnnePdM** [JanMarvin/openxlsx2#408] [Q&A] [answered]
Is there any way to control the window size of the Excel files exported from openxls2? Mine are alway very small. The first thing I need to do when they are opened is maximise the window.

**Comment by JanMarvin** [comment, marked as answer]
Hi @AnnePdM ,

thanks for asking. Unfortunately we do not yet provide an interface for this, but you can tweak 'windowsWidth' and 'windowHeight' this way:

```R
wb$workbook$bookViews <- '<bookViews><workbookView xWindow="0" yWindow="0" windowWidth="13125" windowHeight="6105"/></bookViews>'
```

We use the defaults above, which might originate from `openxlsx` (and non 4k retina displays). I was reminded by an open PR in `openxlsx`: https://github.com/ycphs/openxlsx/pull/383

**Reply by AnnePdM** [reply]
That seems to sort out the problem. Although I am not sure of the units of windowWidth and windowHeight, I put in some sensible sounding numbers and the size was better on both my screens. Thanks!

**Reply by JanMarvin** [reply]
The unit size is [twip](https://en.wikipedia.org/wiki/Twip) and yes I didn't know this either. You can modify a workbook in Excel and save it. Afterwards you can load it with `openxlsx2` this way you can read the value Excel uses with `xml_attr(wb$workbook$bookViews, "bookViews", "workbookView")`. Setting the value to something incredible large simply falls back to full size. So nothing to worry.

**Reply by AnnePdM** [reply]
Ah, maybe I set the value to something too large then because it seemed to maximise - which was actually what I wanted! Thanks for your help.
